src/jojoe77777/FormAPI/FormAPI.php

```diff
@@ -34,7 +34,7 @@ public function createSimpleForm(callable $function = null) : SimpleForm {
      * @param callable|null $function
      * @return ModalForm
      */
-    public function createModalForm(callable $function = null) : ModalForm {
+    public function createModalForm(?callable $function = null) : ModalForm {
```

Comment:
I think it's better to leave these ones as-is. If it annoys devs, that's good because it'll encourage them to stop using deprecated methods.

Reply:
> "If it annoys devs, that's good"

Refusing to read the `@deprecated` tag is expected to have very high correlation with not knowing the problem with `?callable` vs `callable`.

Reply:
I guess you're right. I'll merge this as is.